src/expectation.coffee

```diff
@@ -74,7 +74,19 @@ module.exports = class Expectation
 
     @manager._checkPageSettings(loader.originalOptions)
 
-    _.all ['include', 'only', 'order', 'filters', 'perPage', 'page', 'limit', 'offset', 'search'], (optionType) =>
+    optionKeys = ['include'
+                  'only'
+                  'order'
+                  'filters'
+                  'perPage'
+                  'page'
+                  'limit'
+                  'offset'
+                  'search'
+                  'cacheKey'
```

Comment:
@juanca Hrm, ok.  So `optionalFields` is the only key that's missing from this set?  Do we need to worry about another key being added into `collection` diverging from this set in `expectation` or is brainstem fairly settled?